frontend/.jshintrc

```diff
@@ -0,0 +1,44 @@
+{
+  // Set syntax to ES5 (e.g., complain about use of ES6 features)
+  "esversion": 9,
```

Comment:
This is the only modification I made to the recommended JSHint file here: https://github.com/datamade/node-style-guide/blob/master/.jshintrc This lets us lint the latest JS features.

Reply:
Great!

Reply:
It will likely be necessary to update our style guide, if we decide to use this pattern company-wide. Maybe make a note of that here: https://github.com/datamade/ops/issues/593?